### PR TITLE
Added new variables to BkgDiag

### PR DESCRIPTION
**Pull request by myucel-fnal**

### CODE REVIEW

File: TrkDiag/src/BkgDiag_module.cc [hunk 1]
```diff
@@ -300,9 +309,17 @@ namespace mu2e
...
       bool pce = _pgen==2; // primary from a CE
       _nindex=0;
+      double sumEdep(0.);
+      double sqrSumDeltaTime(0.);
+      double sqrSumDeltaX(0.);
+      double sqrSumDeltaY(0.);
       for(auto const& ich : cluster.hits()){
         ComboHit const& ch = _chcol->at(ich);
         BkgClusterHit const& bhit = _bkghitcol->at(ich);
+        sumEdep +=  ch.energyDep();
```
Comment (brownd1978) [inline TrkDiag/src/BkgDiag_module.cc:319]: edep is measured per straw.  I think this sum should be weighted by the # of straw hits (ch.nStrawHits()

File: TrkDiag/src/BkgDiag_module.cc [hunk 2]
```diff
@@ -300,9 +309,17 @@ namespace mu2e
...
+        sqrSumDeltaX += std::pow(ch.pos().X() - _cpos.X(),2);
```
Comment (brownd1978) [inline TrkDiag/src/BkgDiag_module.cc:320]: The RMS calculation is mathematically correct, but I suspect we will have better discrimination computing the rms of the transverse distance to the straw, not the hit.  That will remove the uncertainty from the position along the straw, and it automatically combines the 2 dimensions.  You can compute that using 2 line POCA, one line being in Z through the cluster position, the other being along the straw (udir) through the hit position.  Or its simple enough to compute analytically.

**Comment by brownd1978** [comment]
@FNALbuild run build test

File: TrkDiag/src/BkgDiag_module.cc [hunk 3]
```diff
@@ -353,6 +370,10 @@ namespace mu2e
         if(bkghinfo._mcpdg == PDGCode::proton)_nprot += ch.nStrawHits();
         _bkghinfo.push_back(bkghinfo);
       }
+      _avecedep = sumEdep/_nchits;
+      _rmscposx = sdt::sqrt(sqrSumDeltaX/_nchits);
```
Comment (kutschke) [inline TrkDiag/src/BkgDiag_module.cc:374]: This line is the build error sdt --> std

**Review by kutschke — CHANGES_REQUESTED** [review]

**Comment by kutschke** [comment]
@myucel-fnal I am checking in.  I think that I am waiting for you to make a commit to fix the build issue and push it to your fork.  Are we on the same page?

**Comment by kutschke** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]

**Comment by kutschke** [comment]
The CI tests passed.

Dave made a suggestion about using a 2 line POCA - it was just a suggestion not a requirement.  Do you plan to incorporate that in this PR - if not then this PR is ready to merge.  Please let me know your plans.

**Comment by myucel-fnal** [comment]
Let me work on those changes, I need to check results to make sure. Hopefully I am finished with it today.

**Pull request merged**